examples/opentelemetry/otel-collector-config.yaml

```diff
@@ -4,6 +4,7 @@ extensions:
   zpages:
     endpoint: 0.0.0.0:55679
   health_check:
+    endpoint: 0.0.0.0:13133
```

Comment:
why is this necessary?

Reply:
I originally did it because the default of the collector will change from binding to 0.0.0.0 to localhost, but then again, the health-check is run from within the container, so there is no actual difference.

Reply:
Reverted in 8e723eb5c8

Reply:
cool, i was curious about this more than anything